plugin.yaml

```diff
@@ -46,3 +92,90 @@ node_types:
             retry_timer:
               type: integer
               default: 10
+
+  cloudify.nodes.bigip.gtm.Connection:
+    derived_from: cloudify.nodes.Root
+    properties:
+      host:
+        type: string
+        description: >
+          IP of the BIG IP GTM instance
+      user:
+        type: string
+        description: >
+          User used to login to BIG IP GTM instance
+      password:
+        type: string
+        description: >
+          Password used to login to BIG IP GTM instance
```

Comment:
Why not have this be a property of the `cloudify.nodes.bigip.gtm.DNSLoadBalancer` node type?